generate-docs.js

```diff
@@ -20,22 +20,30 @@ const isHtmlFile = fileName => fileName.endsWith('.html');
 
 const pipe = (...fns) => start => fns.reduce((state, fn) => fn(state), start);
 
-function hideSearchbar() {
+function adjustmentForGitHubPages() {
   const paths = flatten(readDir('./documentation')).filter(isHtmlFile);
-  paths.forEach(path => replaceSearchbar(path));
-}
+  paths.forEach(path => replaceUnderlinePrefixAndHtmlSuffixFromLinks(path));
+  paths.forEach(path => removeUnderlinePrefixFromFileName(path));
 
-function replaceSearchbar(path) {
-  const labelEl = '<label for="tsd-search-field" class="tsd-widget search no-caption">Search</label>';
-  const inputEl = '<input id="tsd-search-field" type="text" />';
+}
 
+function replaceUnderlinePrefixAndHtmlSuffixFromLinks(path){
   const html = fs.readFileSync(path, { encoding: 'utf8' });
-  if (html.includes(labelEl) && html.includes(inputEl)) {
-    const replaced = html.replace(labelEl, '').replace(inputEl, '');
-    fs.writeFileSync(path, replaced, { encoding: 'utf8' });
-  } else {
-    throw Error('Searchbar could not be hidden. Label and input tags are not defined.');
-  }
+  const replaced = html.replace(/<a href="[^>]*_[^>]*.html">/gi, removeUnderlinePrefixAndHtmlSuffixFromLink);
+  fs.writeFileSync(path, replaced, { encoding: 'utf8' });
+}
+
+function removeUnderlinePrefixAndHtmlSuffixFromLink(aHref) {
+  const i = aHref.indexOf('_');
+  // remove the first `_` and `.html`
+  return aHref.substring(0, i) +  aHref.substring( i+1, aHref.length - 7 ) + '\">'
+}
+
+function removeUnderlinePrefixFromFileName(path){
```

Comment:
here removeUnderlinPrefixAndHtmlSuffixFromFileName would be the correct way.

Reply:
For the file name, the html can stay.
Only the underline has to be removed.